docs/Release_notes.md

```diff
@@ -1,5 +1,139 @@
+- Issue 343: Fix UI - Route Name not showing
+- Issue 348: Autoware plugin does not support plugin discovery
+- Issue 349: Guidance state machine does not transition properly from active to engaged on Lexus
+- Issue 352: CARMA launch file does not provide placeholder to pass map cell paths 
+- Issue 358: CI build fails due to missing ros-kinetic-swri-serial-util pkg
+- Issue 368: Unit test fails in GuidanceStateMachine
+- Issue 370: CARMASscInterfaceWrapper Docker image build failing due to dbw_mkz_msgs 
+- Issue 375: SonarCloud does not report code coverage correctly
+- Issue 381: Fussion CAN topic names mismatch
+- PR 325: Initial implementation of CARMA3 guidance node
+- PR 327: Fix environment variables in launch files.
+- PR 329: Fix argument name for ray_ground_filter 
+- PR 330: Remove utility packages and update checkout.sh to depend on CARMAUtils
+- PR 331: Updating CARMAUtils repo name
+- PR 322: GNSS/NDT auto selector
+- PR 334: Update gnss_to_map_convertor to use new gps api 
+- PR 338: Add enhanced warning flag output to catkin build process
+- PR 339: Update Circle CI for CARMA3
+- PR 371: Update test cases for guidance node
+- PR 374: Created a new Turner Fairbank waypoints
+
+**Repository: autoware.ai**  
+- PR 12: Build new Autoware dockerization system on top of new carma-base
+- PR 17: Add calibration file for ray_ground_filter node
+- PR 18: Update ndt_matching.cpp 
+- PR 22: Disable genjava in Docker build as it causes instability 
+
+**Repository: CARMABase**  
+- PR 7:  Add Autoware dependencies and Sonar Scanner
+- PR 8:  Add code coverage scripts to Docker image
+- PR 10: Update init-env.sh to have proper Autoware install location
+- PR 11: Update SSC Dependencies
+- PR 12: Update for Component Release 3.1.0
+- PR 13: Update package version to 3.1.0 
+
+**Repository: CARMAWebUI**  
+- PR 17: Fix Docker image name 
+- PR 18: Update CircleCI For CARMA 3
+- PR 19: Show Route Name on UI
+
+**Repository: CARMAUtils**  
+- PR 14: Add carma_utils and wgs84_utils packages to repo 
+- PR 15: Update checkout.sh script and CARMAUtils repo name
+- PR 16: Add uncertainty_tools package
+- PR 17: Update CircleCI For CARMA 3
+- PR 18: Feature/enhanced build warnings 
+
+**Repository: CARMAMsgs**  
+- PR 17: Update CircleCI for CARMA 3
+- PR 18: Update Plugin.msg 
+
+**Repository: CARMAConfig** 
+- PR 8: Create initial ford fusion configuration
+- PR 9: Update development config to match parameter standards 
+- PR 10: Remove outdated IMU config in drivers.launch 
+- PR 11: Fix Lexus can configuration
+- PR 12: Add state machine type
+- PR 13: Fix/pacifica configuration
+
+**Repository: CARMAVehicleCalibration** 
+- PR 1: Add initial development and Lexus calibration files
+- PR 2: Add vehicle calibration data
+- PR 3: Update Pacifica yaw offset
+- PR 5: Update blue Lexus LiDAR calibration
+- PR 6: Remove duplicated folders from blue Lexus calibration
+
+**Repository: CARMASscInterfaceWrapper** 
+- PR 13: Update for Docker 
```

Comment:
At least one of these PRs is the result of issue #370 above.  I'm okay leaving this list as is, but for future releases it would be nice to remove the duplicate info.  Ideally, there are no PRs listed in this file, only issues, and that covers everything changed.